TOOLS/appveyor-build.sh

```diff
@@ -1,24 +1,4 @@
-#!/usr/bin/bash
-set -e
+#!/usr/bin/bash -e
 
-export DEST_OS=win32
-export CC=gcc
-export PKG_CONFIG=/usr/bin/pkg-config
-export PERL=/usr/bin/perl
-export PYTHON=/usr/bin/python3
-
-"$PYTHON" bootstrap.py
-"$PYTHON" waf configure \
-    --check-c-compiler=gcc \
-    --enable-spirv-cross \
-    --enable-d3d-hwaccel \
-    --enable-d3d11 \
-    --enable-jpeg \
-    --enable-lcms2 \
-    --enable-libarchive \
-    --enable-lua \
-    --enable-rubberband \
-    --enable-shaderc \
-    --enable-uchardet \
-    --enable-vulkan
-"$PYTHON" waf build
+meson setup build -D libmpv=true
```

Comment:
I think the features were specified explicitly so that the build throws an error if any of them (unexpectedly) go missing

Reply:
Done